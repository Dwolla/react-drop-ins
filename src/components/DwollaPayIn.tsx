import type { FC } from 'react';
import { PayInAttributes } from '../@types/component-attributes';

const DwollaPayIn: FC<PayInAttributes<any>> = (props) => {
  return <dwolla-customer-create {...props} />;
};

export default DwollaPayIn;
