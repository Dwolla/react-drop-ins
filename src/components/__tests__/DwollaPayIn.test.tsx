import TestRenderer from 'react-test-renderer';
import DwollaPayIn from '../DwollaPayIn';

test('DwollaPayIn', () => {
  const tree = TestRenderer.create(<DwollaPayIn customerId="123" blob={''} token={''} />).toJSON();

  expect(tree).toMatchSnapshot();
});

test('with optional props', () => {
  const tree = TestRenderer.create(
    <DwollaPayIn customerId="123" amount={10} blob={''} token={''} />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
