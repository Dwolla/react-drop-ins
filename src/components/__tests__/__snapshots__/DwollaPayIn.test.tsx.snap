// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DwollaPayIn 1`] = `
<dwolla-customer-create
  blob=""
  customerId="123"
  token=""
/>
`;

exports[`with optional props 1`] = `
<dwolla-customer-create
  amount={10}
  blob=""
  customerId="123"
  token=""
/>
`;
